lib/mapper/product_mapper.rb

```diff
@@ -30,7 +30,7 @@ def self.map_payload(product, additional_data = {}, track_inventory = true, defa
                   type: isDigital ? 'digital' : 'physical',
                   weight: product['weight'] ? product['weight']['value'] : '0',
                   width: product['width'] ? product['width']['value'] : '0',
-                  inventory_tracking: isDigital || !track_inventory ? 'none' : 'product',
+                  inventory_tracking: isDigital || !track_inventory || product['nonInventory'] || product['assembly'] ? 'none' : 'product',
```

Comment:
@Jhalter5Stones This agent should only have to check one field now. The acumen agent should set something like `trackInventory` or `shouldTrackInventory`, and this agent should look at that field.

Reply:
@mrogers-5s Fixed.